dev/core/src/com/google/gwt/dev/shell/HostedModePluginObject.java

```diff
@@ -193,8 +191,8 @@ public boolean connect(String url, String sessionKey, String address,
     // Object[] with a success boolean and a value, and HtmlUnit will guard against this.
     // The simplest way to do that here is to mark java.lang.Object as the java equivalent
     // of some JS type - the name of the type doesn't matter.
-    webClient.setActiveXObjectMap(Collections.singletonMap(
-            "GwtLegacyDevModeExceptionOrReturnValue", "java.lang.Object"));
+    // webClient.setActiveXObjectMap(Collections.singletonMap(
+    //        "GwtLegacyDevModeExceptionOrReturnValue", "java.lang.Object"));
```

Comment:
If we're removing this, we should probably remove it outright, and the comment too.

That said, I think legacy dev mode tests work still with HtmlUnit, so this is probably required to keep? Unless we are actually outright removing that feature, in which case there is a lot more to remove too?